### PR TITLE
Backport 1.16 to 1.15

### LINKED ISSUE #186: List of thinks to backport from 1.16 to 1.15, eta: once I will stop being a slacker
- [x] a132598
- [x] 8fe8675

### PR DESCRIPTION
**Pull request by Nightenom**
Closes #186, includes #187

Review please

### CODE REVIEW

File: src/main/java/com/ldtteam/structures/client/BlueprintHandler.java [hunk 1]
```diff
@@ -113,10 +114,10 @@ public void clearCache()
      * @param partialTicks the partial ticks.
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structures/client/BlueprintHandler.java:114]: adjust jdc

File: src/main/java/com/ldtteam/structures/client/StructureClientHandler.java [hunk 2]
```diff
@@ -16,16 +15,29 @@ private StructureClientHandler()
...
     }
-}
+
+    @Deprecated // INTERNAL USE ONLY
+    public static void renderStructure(@NotNull final Blueprint blueprint, final float partialTicks, final List<BlockPos> points, final MatrixStack stack)
+    {
+        BlueprintHandler.getInstance().drawAtListOfPositions(blueprint, points, stack, partialTicks);
+    }
+
+    public static void renderStructureAtPosList(@NotNull final Blueprint blueprint, final float partialTicks, final List<BlockPos> points, final MatrixStack stack)
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structures/client/StructureClientHandler.java:49]: jdoc

File: src/main/java/com/ldtteam/structures/helpers/Settings.java [hunk 3]
```diff
@@ -98,8 +103,31 @@ private Settings()
          */
     }
 
+    public void startStructurizePass()
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structures/helpers/Settings.java:109]: jdoc

File: src/main/java/com/ldtteam/structures/client/BlueprintHandler.java (hunk 1 above)
Comment (Nightenom) [reply src/main/java/com/ldtteam/structures/client/BlueprintHandler.java:114]: hmm?

Comment (Raycoms) [reply src/main/java/com/ldtteam/structures/client/BlueprintHandler.java:114]: jdoc

**Pull request merged**